.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+**Home Assistant release (`hass --version`):**
+
+
+**Python release (`python --version`):**
```

Comment:
In general, if both Python 2 and 3 are installed, `python` defaults to Python 2 and `python3` to Python 3. This is the case for example for all Debian based systems (including Raspbian)